codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/testutil/Rust.kt

```diff
@@ -42,6 +42,43 @@ import java.nio.file.Files.createTempDirectory
 import java.nio.file.Path
 import kotlin.io.path.absolutePathString
 
+// cargo commands and env values
+object Commands {
+    private const val cfgUnstable = "--cfg aws_sdk_unstable"
+    fun func(s: String, add: String, flag: Boolean): String {
+        if (flag) {
+            return s + " " + add
+        } else {
+            return s
+        }
+    }
+
+    fun cargoEnvDWarnings(enableUnstable: Boolean): Map<String, String> {
```

Comment:
Looks like this one isn't denying warnings? Should also rename to `cargoEnvDenyWarnings`.